sku.0/sys.server/compiled/game/script/theme_park/wod/wod_mutant_rancor_combat.java

```diff
@@ -0,0 +1,229 @@
+package script.theme_park.wod;
+
+import script.*;
+import script.base_class.*;
+import script.combat_engine.*;
+import java.util.Arrays;
+import java.util.Hashtable;
+import java.util.Vector;
+import script.base_script;
+
+import script.library.ai_lib;
+import script.library.beast_lib;
+import script.library.buff;
+import script.library.chat;
+import script.library.combat;
+import script.library.create;
+import script.library.group;
+import script.library.prose;
+import script.library.static_item;
+import script.library.trial;
+import script.library.utils;
+
+public class wod_mutant_rancor_combat extends script.base_script
```

Comment:
This script should probably be renamed for clarity. This is not just the *Mutant Rancor* but rather the *Corrupted Rancor Mutation* boss from The Greater Good quest.